src/olympia/devhub/tasks.py

```diff
@@ -106,7 +106,7 @@ def submit_file(addon_pk, upload_pk, channel):
 
 
 @transaction.atomic
-def create_version_for_upload(addon, upload, channel):
+def create_version_for_upload(addon, upload, channel, parsed_data=None):
```

Comment:
😬 I forgot we have this complicated non-task in tasks.py ... should it be move to devhub/utils.py?